content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -169,6 +171,27 @@ CREATE TABLE "Example" (
 CREATE INDEX "Example_val_idx" ON "Example" USING HASH (val);
 ```
 
+
+
+### Configuring the access type of indexes with `type` (PostgreSQL)
+
+The `clustered` argument is available to configure (non)clustered indexes in SQL Server in version `3.13.0` and later. It can be used on `@id`, `@@id`, `@unique`, `@@unique` and `@@index` attributes.
```

Comment:
I find "(non)clustered" phrasing a bit confusing... is that saying that you can set `clustered` to either `false` or `true`?

Reply:
You can define if the index should be clustered or non-clustered. 
The defaults are different between these attributes.

Reply:
OK, happy to go with this phrasing now I understand it

Reply:
I am not super happy about it myself, but I also updated the headline now and maybe that helps a bit.